src/common/service/NetRemoteService.cxx

```diff
@@ -389,6 +424,156 @@ NetRemoteService::WifiAccessPointDisable([[maybe_unused]] ::grpc::ServerContext*
     return grpc::Status::OK;
 }
 
+using Microsoft::Net::Wifi::Dot11PhyType;
+
+::grpc::Status
+NetRemoteService::WifiAccessPointSetPhyType([[maybe_unused]] ::grpc::ServerContext* context, const ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetPhyTypeRequest* request, ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetPhyTypeResult* response)
+{
+    using Microsoft::Net::Wifi::Ieee80211AccessPointCapabilities;
+
+    LOGD << std::format("Received WifiAccessPointSetPhyType request for access point id {}", request->accesspointid());
+
+    WifiAccessPointOperationStatus status{};
+    status.set_code(WifiAccessPointOperationStatusCode::WifiAccessPointOperationStatusCodeSucceeded);
+
+    auto handleFailure = [&](WifiAccessPointOperationStatusCode statusCode, const std::string& statusMessage) {
+        LOGE << statusMessage;
+
+        status.set_code(statusCode);
+        status.set_message(statusMessage);
+
+        response->set_accesspointid(request->accesspointid());
+        *response->mutable_status() = std::move(status);
+
+        return grpc::Status::OK;
+    };
+
+    // Check if PHY type is provided.
+    if (request->phytype() == Dot11PhyType::Dot11PhyTypeUnknown) {
+        handleFailure(WifiAccessPointOperationStatusCode::WifiAccessPointOperationStatusCodeInvalidParameter, "No PHY type provided");

```

Comment:
Do you mean to `return` here? It seems like it since the lambda is returning a `grpc::Status`. Same applies to other instances where `handleFailure` is used in this function.

```suggestion
        return handleFailure(WifiAccessPointOperationStatusCode::WifiAccessPointOperationStatusCodeInvalidParameter, "No PHY type provided");
```

Reply:
Yes you're right, there should be `return`s at each call to `handleFailure`